index.md

```diff
@@ -50,7 +50,7 @@ Want to know more about administering a Jellyfin server? Check out these pages!
 * [Quick-Start Guide](xref:quick-start): What to do after you've installed Jellyfin to get it up and running.
 * [Migrating from Emby](xref:admin-migrate-from-emby): How to migrate an existing Emby 3.5.2 installation to Jellyfin.
 * [Plugins](xref:server-plugins-index): How to install and manage plugins.
-* [Port Bindings](xref:admin-port-bindings): All ports used when running the server.
+* [Networking](xref:network-index): Networking settings and troubleshooting
```

Comment:
```suggestion
* [Networking](xref:network-index): Networking settings and troubleshooting.
```

Reply:
So the xref's kind of annoy me since I can't click them on github.